RNG.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+print ("Welcome to the random name generator!")
+
+def get_shortlist():
+# Reads the names that th user inputs and returns a list.
+    shortlist_str = input("Please enter names, separated by commas: ")
+    return [name.strip() for name in shortlist_str.split(',')]
+    
+def get_num_names():
+# Reads the number of names to be selected from the list and returns the interger.
+  while True: 
+    num_names_str = input("Enter the number of names to be selected: ")
+    if num_names_str.isdigit():
+        return int(num_names_str)
+    else:
+            print("Invalid input. Please Enter a postive interger.")
```

Comment:
Good short and concise function! Very easy to understand the whole function.
Just a question, does the function stop getting inputs if 'else' condition is triggered?
Does the 'if' condition consider negative values?

Reply:
In regards, to your question about the 'else' function, it will just tell you to "Enter the number of names to be selected:" again. The 'if' function does consider negative values when ran since the code only asks for positive integers and will pop up a message.